drake/systems/plants/xmlUtil.cpp

```diff
@@ -205,7 +205,25 @@ string resolveFilename(const string& filename,
       return string();
     }
   } else {
-    mesh_filename_s = spruce::path(root_dir);
+    std::string normalized_root_dir = spruce::path(root_dir).getStr();
+
+    // if root_dir is a relative path then convert it to absolute
+#ifdef _WIN32
+    bool dirIsRelative = !(normalized_root_dir.size() >= 2
+                           && std::isalpha(normalized_root_dir[0])
```

Comment:
This is the only line that actually introduces something new, a reference to the std::isalpha function.  The rest of the branch is just std string manipulation.